src/templates/React InstantSearch Native/app.json

```diff
@@ -1,7 +1,6 @@
 {
   "expo": {
     "name": "{{name}}",
-    "slug": "{{name}}",
-    "sdkVersion": "32.0.0"
+    "slug": "{{name}}"
```

Comment:
Why do we no longer need `sdkVersion`?

Reply:
It wasn't set in a default expo CLI app, but I added it back in b76d49b with the value set to the correct Expo SDK version.